cmd/apiserver/main.go

```
@@ -24,65 +24,115 @@ import (
+func run(ctx context.Context, args []string) error {
+	ctx, cancel := signal.NotifyContext(ctx, os.Interrupt)
+	defer cancel()
+
+	// change here: if you want to use a different logger
+	log, err := logger.NewZapLogger()
+	if err != nil {
+		return err
+	}
+
+	if flushable, ok := log.(logger.Flushable); ok {
+		defer flushable.Sync() // ensure flush, for flushable loggers
```

golangci-lint (1.62.2)

Error return value of `flushable.Sync` is not checked

(errcheck)